pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -315,6 +319,9 @@ func ConfigPath(ctx *types.SystemContext) string {
 			confPath = filepath.Join(ctx.RootForImplicitAbsolutePaths, systemRegistriesConfPath)
 		}
 	}
+	if _, err := os.Stat(filepath.Join(homedir.Get(), userRegistriesFile)); err == nil {
```

Comment:
I'd suggest defining a new variable, perhaps secureConfPath and set it using  securejoin.SecureJoin() before doing the os.Stat then just return it.

Reply:
Why `SecureJoin`? We know what `userRegistriesFile` is, and that it won’t try to escape from the home directory.

Reply:
It _would_ be useful, though, to have the `filepath.Join()` computation not repeated twice.

Reply:
"Why SecureJoin?" too many burn marks.  You're right, should be good with filepath.join, but would like to see whichever join done just once.